neofetch

```diff
@@ -7346,6 +7346,41 @@ dMm     `/++/-``/yNNh+/sdNMNddMm-    mMd
       /dMMh+.              .+hMMd/
         -odMMNhso//////oshNMMdo-
            `:+yhmNMMMMNmhy+:`
+EOF
+        ;;
+
+        "EshanizedOS"*)
+            set_colors 1 7
+            read -rd '' ascii_data <<'EOF'
+${c1}            .:-==++++++++++++++++++++++-.  
```

Comment:
Please pad with spaces to make the first line align.

![image_2023-12-01_14-40-55](https://github.com/dylanaraps/neofetch/assets/22280294/676d5bd2-9cd4-4c0f-8378-a5f89d2f31ce)